src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -2,10 +2,20 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
 
+@Entity(name = "movies")
```

Comment:
it is better to use `@Table` annotation for this purpose
we will see why when we will write our custom queries

please change it for all 3 entities